### PR TITLE
refactor: MSP binding to `this`

### PR DESCRIPTION
**Pull request by chmelevskij**
Part of work moving configurator to web:
* remove `self` pattern
* use arrow functions for proper binding to `this`
* use method syntax for MSP methods

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by asizon — APPROVED** [review]
Good Code clean!

**Pull request merged**
